### ISSUE TITLE
Home screen widget

### ISSUE THREAD

**Issue by jcrabapple** [DarkMooNight/ToDark#8]
I would love a home screen widget or a "create task" activity so I can make a custom shortcut to create a new task.

**Comment by Leonavichus** [comment]
Ok, I will try to implement it, but it may take time, as there is a lot of other work.

**Comment by jcrabapple** [comment]
Thank you for taking my feedback into consideration.

**Comment by danielyrovas** [comment]
Yes, widgets are awesome, especially if the todo list can be viewed and edited without opening the application.

**Comment by OwenD30** [comment]
I would like to add support for this request. The lack of a widget where I can view my to dos and check them off is the one thing keeping me using Google tasks. 

Amazing app in all other respects!!!

**Comment by Avichal1234** [comment]
Hi, is there any update on the progress of this?
